genie-web/src/test/groovy/com/netflix/genie/web/tasks/job/JobCompletionServiceSpec.groovy

```diff
@@ -149,10 +146,8 @@ class JobCompletionServiceSpec extends Specification {
                 .withId(jobId).withStatus(JobStatus.RUNNING).withCommandArgs(COMMAND_ARGS).build()
         1 * jobPersistenceService.updateJobStatus(jobId, _, _)
         1 * completionTimer.record(_, TimeUnit.NANOSECONDS)
-        timerTagsCapture == ImmutableMap.of(
-                MetricsConstants.TagKeys.STATUS, MetricsConstants.TagValues.SUCCESS,
-                JobCompletionService.JOB_FINAL_STATE, JobStatus.FAILED.toString()
-        )
+        verifySuccessTags(timerTagsCapture) \
+         && timerTagsCapture.get(JobCompletionService.JOB_FINAL_STATE).equals(JobStatus.FAILED.toString())
```

Comment:
No need to AND
Place one condition you expect to return `true` per line, this produces the most clear output when one of the assertions fails.